client/packages/host/public/config.local.js

```diff
@@ -0,0 +1,3 @@
+window.env = {
+  API_HOST: 'https://localhost:8000', // - default URL for the backend graphql server
+};
```

Comment:
Cool, maybe remove the commented-out one from the original `config.js`?

Reply:
was leaving it in as that's required when building for android it requires the local version.
However.. your comment prompted c097a2b09 which is to do the config change with scripts and is much easier! thanks.